spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "Studios", type: :feature do
+  describe "Studio Index" do
+    it "displays a list of all studios" do
+    studio1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    movie1 = Movie.create!(title: "Toy Story", creation_year: "1995", genre: "Animation", studio: studio1)
+    movie2 = Movie.create!(title: "The Godfather", creation_year: "1972", genre: "Crime", studio: studio1)
+    visit "/studios"
+
+    expect(page).to have_content(studio1.name)
+    expect(page).to have_content(studio1.location)
+    expect(page).to have_content(movie1.title)
+    expect(page).to have_content(movie1.creation_year)
+    expect(page).to have_content(movie1.genre)
```

Comment:
Since we want to make sure this info appears for ALL movies, I would also create `expect` lines for movie2, to ensure we see them all and not just the first one.